### DISCUSSION TITLE
[Bug 2.0.0-alpha.5] Webp images from external sources/URLs are not cached nor displayed

### DISCUSSION THREAD

**Discussion by aitoraroza** [automadcms/discussions#14] [General]
Hello, there.
Cache does not create images in webp format from external sources. They are accepted both in backend live editor and in JSON page data file, which stores URLs correctly. As a result, no image or html img tag outputs at webpage render. Besides, even if cache is disabled in system conf, it's still forced/enabled for images.

I've noticed that webp images directly stored within automad ("shared" folder) are properly processed in cache and shown afterwards. So maybe is something really easy to solve.

Any idea for a quick fix? I would like to use another server/CDN as image source, to keep things tidy.

Thanks in advance, and kudos for this powerful cms. I'm almost decided to give a try and move a site from getgrav to automad! :D

**Comment by marcantondahmen** [comment]
Hi, thanks for reporting this.

**Comment by marcantondahmen** [comment]
This should be fixed now with the latest alpha 6.
